makecrx.sh

```diff
@@ -33,25 +33,11 @@ fi
 
 VERSION=`python2.7 -c "import json ; print(json.loads(open('chromium/manifest.json').read())['version'])"`
 
-echo "Building chrome version" $VERSION
+echo "Building crx version" $VERSION
```

Comment:
This comment has been updated in https://github.com/EFForg/https-everywhere/tree/webextensions already.